nautobot/utilities/forms/fields.py

```diff
@@ -458,3 +459,96 @@ def prepare_value(self, value):
         if value is None:
             return ""
         return json.dumps(value, sort_keys=True, indent=4)
+
+
+class JSONArrayFormField(forms.JSONField):
+    """
+    A FormField counterpart to JSONArrayField.
+    Replicates ArrayFormField's base field validation: Field values are validated as JSON Arrays,
+    and each Array element is validated by |base_field| validators.
```

Comment:
If you could sub the `|foo|` pipes w/ backticks here in this class, too, please.